### PR TITLE
Mlflow docs patch

### PR DESCRIPTION
**Pull request by dennyglee**
**PR Checklist**

- [X] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
Minor formatting pages to the docs/quickstart including the MLflow section.

### CODE REVIEW

**Review by kriscon-db — APPROVED** [review]
LGTM on the models change.

File: docs/quickstart.md [hunk 1]
```diff
@@ -6,38 +6,105 @@ This quickstart shows how to run Unity Catalog on localhost which is great for e
...
+│294                                    │209.32243623208947                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│150                                    │329.19730274053694                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│539                                    │425.66102859000944                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│247                                    │477.742227230588                      │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│958                                    │509.3712727285101                     │
+└───────────────────────────────────────┴──────────────────────────────────────┘
```
Comment (tdas) [inline docs/quickstart.md:105]: can we truncate these tables with `...` 
just a few rows prove the point.

File: docs/quickstart.md [hunk 2]
```diff
@@ -88,63 +161,223 @@ you have to list different asset types separately. Let's start with tables.
...
+│UPDATED_AT                     │1721266805617                                                                                                                                │
+├───────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│UPDATED_BY                     │null                                                                                                                                         │
+├───────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│TABLE_ID                       │32025924-be53-4d67-ac39-501a86046c01                                                                                                         │
+└───────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```
Comment (tdas) [inline docs/quickstart.md:230]: maybe truncate

File: docs/quickstart.md [hunk 3]
```diff
@@ -88,63 +161,223 @@ you have to list different asset types separately. Let's start with tables.
...
+│294                                    │209.32243623208947                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│150                                    │329.19730274053694                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│539                                    │425.66102859000944                    │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│247                                    │477.742227230588                      │
+├───────────────────────────────────────┼──────────────────────────────────────┤
+│958                                    │509.3712727285101                     │
+└───────────────────────────────────────┴──────────────────────────────────────┘
```
Comment (tdas) [inline docs/quickstart.md:274]: same

File: docs/quickstart.md [hunk 4]
```diff
@@ -88,63 +161,223 @@ you have to list different asset types separately. Let's start with tables.
...
+│UPDATED_AT                     │1727374971317                                                                                                                                │
+├───────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│UPDATED_BY                     │null                                                                                                                                         │
+├───────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│TABLE_ID                       │263a234a-7a29-44c8-8e41-8c46aa30650c                                                                                                         │
+└───────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```
Comment (tdas) [inline docs/quickstart.md:320]: same

File: docs/quickstart.md [hunk 5]
```diff
@@ -224,7 +456,14 @@ result[:4]
 
 This code snippet will create a registered model `default.unity.iris` and log the trained model as model version 1.  It then loads the model from the Unity Catalog server, and performs batch inference on the test set using the loaded model.
 
+The results can be seen in the Unity Catalog UI at [http://localhost:3000,](http://localhost:3000) per the instructions in the [Interact with the Unity Catalog tutorial](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#interact-with-the-unity-catalog-ui).
```
Comment (tdas) [inline docs/quickstart.md:459]: shouldnt we add this for tables as well?

**Review by tdas — APPROVED** [review]
just a bunch nits. can be handled in a different pr as well.

**Comment by dennyglee** [comment]
Thanks @tdas - created #516

**Pull request merged**